packages/forms-web-app/src/validators/common/schemas/file-upload.js

```diff
@@ -46,7 +47,8 @@ const schema = (noFilesError) => ({
 
 				//check file for virus
 				const { name } = req.files[path];
-				await validAV(req.files['file-upload'], name);
+				const clamAVClient = new ClamAVClient(clamAVHost, uploadApplicationMaxFileSize);
```

Comment:
Could this be constructed outside the schema function?

Reply:
yes, I guess the max request size should probably move to a param for scan as well, will do that now

Reply:
So I tried this but it led to 30+ unrelated test suites breaking which required either a load of mocking or a workaround, so have undone this change
might look into tests another time
I don't think creating this class on each request is that inefficient, or are we potentially losing connection pooling somewhere along the line by not reusing the same instance?

Reply:
Given we just call axios in the class function, I don't think we miss much. 

We could have a function that returns the singleton/instance, and only call that where it's needed - shouldn't break other tests then? May not be worth it!

Reply:
Have done the require within a function call